scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
global dependencies are in system.sh - but the issue is not the size, but the delay to check each dependency (it's already a noticeable startup delay due to this. But it can be added sure.